tracer/src/Datadog.Trace/AppSec/ControllerContextExtensions.Framework.cs

```diff
@@ -88,7 +88,12 @@ internal static void MonitorBodyAndPathParams(this IControllerContext controller
 
                     if (pathParamsDic.Count > 0)
                     {
-                        inputData.Add(AddressesConstants.RequestPathParams, ObjectExtractor.Extract(pathParamsDic));
+                        var pathParams = ObjectExtractor.Extract(pathParamsDic);
+
+                        if (pathParams is not null)
```

Comment:
was the bug around here? if `pathParams` is null when `pathParamsDic` contained items maybe we should log something cause this is quite strange, means the `ObjectExtractor` failed somewhere

Reply:
The original error was in object extractor: var item = ExtractType(body.GetType(), body, 0, visited);

body was null sometimes